CRM/Price/BAO/PriceSet.php

```diff
@@ -430,11 +430,12 @@ public static function getAssoc($withInactive = FALSE, $extendComponentName = FA
    *   Appears to have no effect based on reading the code.
    * @param bool $validOnly
    *   Should only fields where today's date falls within the valid range be returned?
+   * @param bool $resetOptions
```

Comment:
This parameter is introduced for unit test which needs to reload price options when sale tax is enabled, in order to get tax_amount and tax_rate info.

Reply:
I think we should switch to using \Civi::$statics inside the CRM_Price_BAO_PriceField::getOptions rather than add this param if it is to support unit tests - we have been doing that elsewhere